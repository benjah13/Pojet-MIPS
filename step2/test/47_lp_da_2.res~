Table des symboles .text .data .bss .reginfo .pdr X Tab 
------ Fichier ELF "./res/mult.o" : sections lues lors du chargement ------

 00000000:	21 08 00 11 
 00000004:	21 08 00 12 


<X>
 00001000:	00 00 0a bc 


<Tab>
 00002000:	00 00 00 00 
 00002004:	00 00 00 00 
 00002008:	00 00 00 00 

Command to execute : ADDI $8, $8, 17

Command to execute : ADDI $8, $8, 18

Command to execute : NOP 

Command to execute : BEQ $0, $0, 0

Command to execute : SRL $1, $0, 10

Command to execute : SW $20, 28855(4)

Command to execute : SW $31, 65535(31)

Command to execute : BEQ $0, $0, 0

Command to execute : SRL $1, $0, 10

Command to execute : SW $20, 28855(4)

