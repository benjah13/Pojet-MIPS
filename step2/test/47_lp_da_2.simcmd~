#Commande a lancer dans le script 47_lp_da_2

lp mu
da 0:10
