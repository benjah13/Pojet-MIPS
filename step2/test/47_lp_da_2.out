Table des symboles .text .data .bss .reginfo .pdr EXIT mult reussi 
------ Fichier ELF "./res/mult.o" : sections lues lors du chargement ------

 00000000:	20 19 00 78 
 00000004:	20 0a ff d5 
 00000008:	20 0b 00 ff 
 0000000c:	15 60 00 02 
 00000010:	01 4b 00 1a 
 00000014:	00 07 00 0d 
 00000018:	24 01 ff ff 
 0000001c:	15 61 00 04 
 00000020:	3c 01 80 00 
 00000024:	15 41 00 02 
 00000028:	00 00 00 00 
 0000002c:	00 06 00 0d 
 00000030:	00 00 50 12 
 00000034:	00 00 40 12 
 00000038:	00 00 48 10 
 0000003c:	11 20 00 01 
 00000040:	08 00 00 1e 
<mult>
 00000044:	00 00 00 00 
 00000048:	00 00 00 00 
 0000004c:	01 0b 00 18 
 00000050:	00 00 40 12 
 00000054:	00 00 48 10 
 00000058:	00 00 00 00 
 0000005c:	00 00 00 00 
 00000060:	00 00 68 20 
 00000064:	15 2d 00 04 
 00000068:	11 0a 00 01 
 0000006c:	08 00 00 1e 
<reussi>
 00000070:	20 0a 00 01 
 00000074:	08 00 00 1e 
<EXIT>
 00000078:	00 00 00 0c 


 00001000:	0c aa bb cc 
 00001004:	dd ff 00 00 
 00001008:	aa bb cc dd 



00000000:	ADDI	$t9, $zero, 120
00000004:	ADDI	$t2, $zero, -43
00000008:	ADDI	$t3, $zero, 255
0000000c:	BNE	$t3, $zero, 2
00000010:	DIV	$t2, $t3 
00000014:	Unknown instruction
00000018:	Unknown instruction
0000001c:	BNE	$t3, $at, 4
00000020:	LUI	$at, -32768
00000024:	BNE	$t2, $at, 2
